sigs.yml

```
@@ -0,0 +1,502 @@
+        name: '#otel-sampling'
+        id: C027DS6GZD3
+    invites: calendar-spec-sampling
+    tcSponsors:
+    - name: Josh MacDonald
+      github: jmacd
+    gcLiaison: []
+  - name: 'Specification: Configuration'
+    meeting: Every other Monday at 8:00 PT
+    notes:
+      type: gDoc
+      value: 1uNgWQLQZcSVfBLRXfW9XCVJmKa5NH9R15fSOXhmpGWw
+    chat:
+      - type: slack
+        name: '#otel-config-file'
+        id: C0476L7UJT1
+    invites: calendar-spec-config
+    tcSponsors:
+    - name: Carlos Alberto
+      github: carlosalberto
+    - name: Jack Berg
+      github: jack-berg
+    gcLiaison: []
+  - name: 'Specification: Events'
+    meeting: Every Friday at 10:00 AM PT
+    notes:
+      type: gDoc
+      value: 1BKjQWP32FXL9g1cGbyj7DMXV1Uq_RL8_78rWaMBhN0A
+    chat:
+      - type: slack
+        name: '#otel-event'
+        id: C062HUREGUV
+    invites: calendar-spec-events
+    tcSponsors: []
+    gcLiaison: []
+  - name: 'Semantic Conventions: General'
+    meeting: Every Monday at 08:00 PT
+    notes:
+      type: gDoc
+      value: 10xG7DNKWRhxNmFGt3yYd3980a9uwS8lMl2LvQL3VNK8
+    chat:
+      - type: slack
+        name: '#otel-semantic-conventions-wg'
+        id: C041APFBYQP
+    invites: calendar-semconv
+    tcSponsors:
+    - name: Armin Ruech
+      github: arminru
+    - name: Josh Suereth
+      github: jsuereth
+    - name: Reiley Yang
+      github: reyang
+    gcLiaison: []
+  - name: 'Semantic Conventions: Messaging'
+    meeting: Every Thursday at 8:00 PT
+    notes:
+      type: gDoc
+      value: 1dWHhyXnfVife-cQ2DW5-d5Ldp1Lq8Rre2UsHpyo8cEE
+    chat:
+      - type: slack
+        name: '#otel-messaging'
+        id: C02Q4AAHDSA
+    invites: calendar-semconv
+    tcSponsors: []
+    gcLiaison: []
+  - name: 'Semantic Conventions: System Metrics'
+    meeting: Every Thursday at 08:30 PT
+    notes:
+      type: gDoc
+      value: 1p5TH57t43XpxA48onLzX4PIr3g6ydYKCtR_AUlsCnQk
+    chat:
+      - type: slack
+        name: '#otel-system-metrics-wg'
+        id: C05CTFE9U4A
+    invites: calendar-semconv
+    tcSponsors:
+    - name: Josh Suereth
+      github: jsuereth
+    gcLiaison: []
+  - name: 'Semantic Conventions: LLM'
+    meeting: Every Wednesday alternating between 08:00 and 16:00 PT
+    notes:
+      type: gDoc
+      value: 1EKIeDgBGXQPGehUigIRLwAUpRGa7-1kXB736EaYuJ2M
+    chat:
+      - type: slack
+        name: '#otel-llm-semconv-wg'
+        id: C06KR7ARS3X
+    invites: calendar-semconv
+    tcSponsors:
+    - name: Liudmila Molkova
+      github: lmolkova
+    gcLiaison: 
+    - name: Alolita Sharma
+      github: alolita
+- name: Implementation SIGs
+  sigs:
+  - name: Maintainer Meeting
+    meeting: Every Monday at 09:00PT
+    notes:
+      type: gDoc
+      value: 1_X2CTiDXmxk2JozBNwQ2uP8FfV1rxx9ISE5puHu0yIk
+    chat:
+      - type: slack
+        name: '#otel-maintainers'
+        id: C01NJ7V1KRC
+    invites: calendar-maintainer-meeting
+    tcSponsors: []
+    gcLiaison: []
```

```suggestion
    gcLiaison: []
```
@open-telemetry/governance-committee please assign yourself for being the liaison person for this SIG

---

do we want sponsors/liaison for maintainer meeting?

---

good question, I would say no

---

```suggestion
```

(Not to myself: I might need to update the code in a way that it accepts this)

---

Ehh, I'll do it, I host it most weeks anyways

---

```suggestion
    gcLiaison:
       - name: Morgan McLean
          github: mtwo
```